.ci_support/linux_aarch64_.yaml

```diff
@@ -13,7 +13,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-aarch64
+- quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
That is because this is being cross-compiled. Meaning the build happens on `x86_64` and targets `aarch64`

Moving it off on Travis CI as there have sometimes been issues with that provider. Moving to Azure means the builds need to run on `x86_64` (hence cross-compilation)

Reply:
I see! Thanks for the explanation!

Reply:
Ofc can understand why this would be confusing. It's good to check 🙂